cluster-operator/src/test/java/io/strimzi/operator/cluster/TestUtils.java

```diff
@@ -40,7 +39,7 @@ public static ConfigMap getJmxMetricsCm(String data, String metricsCMName) {
                 .withNewMetadata()
                 .withName(metricsCMName)
                 .endMetadata()
-                .withData(singletonMap(AbstractModel.ANCILLARY_CM_KEY_METRICS, data))
+                .withData(singletonMap("metrics-config.yml", data))
```

Comment:
Wouldn't be better to introduce some constant for the file name?

Reply:
I guess this relateds to the `getJmxPrometheusExporterMetrics` method above it. Maybe it should be a method parameter?

Reply:
I have defined a variable with `metrics-config.json` in the `Constant' Interface also, so if we want to use a constant instead of passing the parameter we can do that also